### ISSUE TITLE
Remove suggestion that you can "follow users" when you are not logged in and view a profile

### ISSUE THREAD

**Issue by lemmygod** [LemmyNet/lemmy-ui#1480]
### Requirements

- [X] Is this a feature request? For questions or discussions use https://lemmy.ml/c/lemmy_support
- [X] Did you check to see if this issue already exists?
- [X] Is this only a feature request? Do not put multiple feature requests in one issue.
- [ ] Is this a server side (not related to the UI) issue? Use the [Lemmy back end](https://github.com/LemmyNet/lemmy) repo.

### Is your proposal related to a problem?

When you are not logged in and go to a user profile, it says the following:

> You are not logged in. If you use a Fediverse account that is able to follow users, you can follow this user

### Describe the solution you'd like.

Change it to something else or remove it.

### Describe alternatives you've considered.

none

### Additional context

![image](https://github.com/LemmyNet/lemmy-ui/assets/137420192/b17a000e-6b6c-4a10-9ffe-61893726b467)

**Comment by dnswrsrx** [comment]
Hello! I'm up for trying to get a PR in for this issue, but I was wondering what the reasoning is for removing this message. I assume it's because as a Lemmy user, there's no actual way to follow another user?

**Comment by dredshep** [comment]
Yes, you cannot follow users on Lemmy currently. If it is implemented later then of course we should have the message.

**Comment by Nutomic** [comment]
The message is clear enough, it specifically says " If you use a Fediverse account that is able to follow users"
